### DISCUSSION TITLE
Download speeds

### DISCUSSION THREAD

**Discussion by Grungehead93** [bucanero/pkgi-ps3#60] [Q&A] [answered]
Is there anyway to increase my speeds of downloads as am only getting under 4000kb?

Thanks
JK

PS love the pkgi man regardless! 👍

**Comment by bucanero** [comment]
you can try using the `background download` mode that uses the PS3 internal download manager. That's probably the best speed you could actually get on a ps3.

Note: When you add files to the background download queue, you need to restart your PS3 to actually make the PS3 to download the files.

**Comment by mnebarrita** [comment]
Imma slide in the conversation for a lil bit, i just jailbreaked my ps3 not too long ago and i wanted to listen to some music or even watch videos while downloading pkgi files does anyone have a tutorial on how to activate ps3 bg downloads?

Wrong place to ask a question I know I'm sorry (also a noob sorry)

**Comment by bucanero** [comment, marked as answer]
> Imma slide in the conversation for a lil bit, i just jailbreaked my ps3 not too long ago and i wanted to listen to some music or even watch videos while downloading pkgi files does anyone have a tutorial on how to activate ps3 bg downloads?

To enable background download:

- press `Triangle` to open settings
- change "DL Mode" to Background mode
-  press `Triangle` to save settings

next, when you select an item, it will be queued for background download. As noted above, when you add files to the background download queue, you need to restart your PS3 to actually make the PS3 start downloading the files.
